gdoc-import/src/main/java/no/dcat/gdoc/GdocController.java

```diff
@@ -159,6 +159,50 @@ String runConvert() throws Exception {
 
         File dir = new File(converterResultDir);
 
+        File found = getFile(versionId, dir);
+
+        if (found == null) {
+            logger.warn("Not found: " + versionId);
+            result = new ResponseEntity<>(versionId, HttpStatus.NOT_FOUND);
+        } else {
+            logger.info("return " + found.getName());
+            result = getFileContent(found);
+        }
+
+        logger.info(GET_A_VERSION + " used " + (System.currentTimeMillis() - startTime));
+        return result;
+    }
+
+
+    /**
+     * Reads the file and drops it to a ResponseEntity.
+     *
+     * @param found the file to read
+     * @return response with the file as text content.
+     */
+    private ResponseEntity<String> getFileContent(File found) {

```

Comment:
ResponseEntity<String> result;
        try (Stream<String> stream = Files.lines(found.toPath())) {
            StringBuilder sb = new StringBuilder();
            stream.forEach(line -> sb.append(line).append(System.lineSeparator()));
            return new ResponseEntity<>(sb.toString(), HttpStatus.OK);
        } catch (IOException e) {
            logger.error("Unable to retrieve file {}",e.getMessage(),e);
            return new ResponseEntity<>(e.getMessage(), HttpStatus.INTERNAL_SERVER_ERROR);
        }